index.bs

```diff
@@ -10331,25 +10331,27 @@ Note: Remember that interface objects for callback interfaces only exist if they
 when they do exist, they are not [=function objects=].
 
 
-<h5 id="es-interface-call">Interface object \[[Call]] method</h5>
+<h5 id="es-constructible-interfaces" oldids="es-interface-call">Constructible Interfaces</h5>
 
-If the [=interface=] is declared with a
-[{{Constructor}}] [=extended attribute=],
-then the [=interface object=]
-can be called as a function to create an object that implements that
-interface.  Interfaces that do not have a constructor will throw
-an exception when called as a function.
+If the [=interface=] is declared with a [{{Constructor}}] [=extended attribute=],
+then the [=interface object=] can be called as a constructor
+to create an object that implements that interface.
+Calling that interface as a function will throw an exception.
 
-The internal \[[Call]] method
-of the interface object behaves as follows, assuming
-|arg|<sub>0..|n|−1</sub> is the list
-of argument values passed to the constructor, and |I|
-is the [=interface=]:
+Interfaces that do not have a constructor will throw
+an exception when called.
+
+When evaluating the [=function object=] |F|,
+which is the interface object for a given non-callback [=interface=] |I|,
+assuming |arg|<sub>0..|n|−1</sub> as the list of argument values passed |F|,
+and |newTarget| as the NewTarget value, the following steps must be taken:
 
 <ol class="algorithm">
     1.  If |I| was not declared with a [{{Constructor}}]
         [=extended attribute=], then
         <a lt="es throw">throw a <emu-val>TypeError</emu-val></a>.
+    1.  If |newTarget| is <emu-val>undefined</emu-val>, then
```

Comment:
Well, it gets [passed](https://github.com/tobie/webidl/blob/8640fb704c1c60f87db84bc8592b213501574cf2/index.bs#L10347) to the algorithm, no? So that's just its local name.